apps/docsite/sidebars.ts

```diff
@@ -1,5 +1,5 @@
-import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
-
+import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
+import changelog from "../../docs/changelogSplits/changelogOrder.json";
```

Comment:
this will break till a seed change log order file is created

Reply:
We can either merge and run the workfloe to generate the seed or manually push one

Reply:
lets try running the workflow after merging